packages/lib-classifier/src/store/ResourceStore.js

```diff
@@ -5,10 +5,10 @@ import Resource from './Resource'
 
 const ResourceStore = types
   .model('ResourceStore', {
-    active: types.maybe(types.reference(Resource)),
-    headers: types.maybe(types.frozen({
+    active: types.safeReference(Resource),
+    headers: types.optional(types.frozen({
       // etag: types.string // setting this is causing the store to be set with a MST type and thus defined. Maybe a bug?
```

Comment:
I don't think this is a bug - with `frozen`, you can pass an object to provide a default snapshot value, or a subtype, but not a model. See https://github.com/mobxjs/mobx-state-tree/blob/master/packages/mobx-state-tree/src/types/utility-types/frozen.ts#L70

You _could_ just make `headers` a model...

Reply:
Good point. I think I left that comment when I was still confused about the difference between frozen types and model types.